package.json

```diff
@@ -45,19 +43,25 @@
   "atom-shell-version": "0.21.3",
   "virtualbox-version": "4.3.24",
   "virtualbox-filename": "VirtualBox-4.3.24.pkg",
```

Comment:
Great, you're using VirtualBox 4.3.26 on Windows. In another PR we should update from 4.3.24 to 4.3.26 on Mac as well.